deletefb/tools/chrome_driver.py

```diff
@@ -0,0 +1,111 @@
+import re
+import zipfile
+import os, sys, stat, platform
+from urllib.request import urlretrieve
+
+from clint.textui import puts, colored
+import progressbar
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+from .common import NO_CHROME_DRIVER
+
+
+def extract_zip(filename):
+    """
+    Uses zipfile package to extract a single zipfile
+    :param filename:
+    :return: new filename
+    """
+    try:
+        file = zipfile.ZipFile(filename, 'r')
+    except FileNotFoundError:
+        puts(colored.red(f"{filename} Does not exist"))
+        sys.exit()
+
+    # Save the name of the new file
+    new_file_name = file.namelist()[0]
+
+    # Extract the file and make it executable
+    file.extractall()
+
+    driver_stat = os.stat(new_file_name)
+    os.chmod(new_file_name, driver_stat.st_mode | stat.S_IEXEC)
+
+    file.close()
+    os.remove(filename)
+    return new_file_name
+
+
+def setup_selenium(driver_path, options):
+    # Configures selenium to use a custom path
+    driver = webdriver.Chrome(executable_path=driver_path, options=options)
+    return driver
+
+
+def get_webdriver():
+    """
+     Ensure a webdriver is available
+     If Not, Download it.
+    """
+    cwd = os.listdir(os.getcwd())
+    webdriver_regex = re.compile('chromedriver')
+    web_driver = list(filter(webdriver_regex.match, cwd))
+
+    if web_driver:
+        # check if a extracted copy already exists
+        if not os.path.isfile('chromedriver'):
+            # Extract file
+            extract_zip(web_driver[0])
+
+        return os.getcwd() + '/chromedriver'
```

Comment:
I'd prefer something like `return "{0}/chromedriver".format(os.getcwd())` just for sylistic reasons